### ISSUE TITLE
Suit sensors TGUI occasionally steals focus which causes you to flail around

### ISSUE THREAD

**Issue by nikothedude** [Bubberstation/Bubberstation#1820]
Reporting client version: 515.1637

## Issue Summary

Title. Youll be typing, then suddenly your textbox unfocuses and you start bumbling around and you have to click it again.

## Round ID:
4925

## Testmerges:
- [Public Logging Update](https://github.com/Bubberstation/Bubberstation/pull/1545)
- [Massive Overhaul of the Dauntless (Ready for review) (Don't merge until after review so I can update the space version)](https://github.com/Bubberstation/Bubberstation/pull/1633)
- [[TM Only] Nerfs blade ascension, removes Moon+Void ascension](https://github.com/Bubberstation/Bubberstation/pull/1581)
- [Nerfs bloodsucker blood gain from the blood reagent](https://github.com/Bubberstation/Bubberstation/pull/1684)
- [GAS 2 - Part 2 - The Sequel, the worm, what you have been waiting for](https://github.com/Bubberstation/Bubberstation/pull/1471)
- [Bloodsucker power refactor](https://github.com/Bubberstation/Bubberstation/pull/1458)
- [ [MDB Ignore] [IDB Ignore] Bubberstation x Effigy Promo [TM ONLY]](https://github.com/Bubberstation/Bubberstation/pull/1724)
- [The offstation cargo update (Test merge ready)](https://github.com/Bubberstation/Bubberstation/pull/1764)
- [Station power is finally stable and intelligent, can you believe it? [TM ONLY]](https://github.com/Bubberstation/Bubberstation/pull/1816)

## Reproduction:

Just keep it open all round. You'll see what I mean.

**Comment by Majkl-J** [comment]
did you know tgui runs on internet explorer? Anyhow it losing focus is a long known problem with any menu. Might be fixed with 516 since in that byond moves html renders to chromium

**Comment by nikothedude** [comment]
> did you know tgui runs on internet explorer? Anyhow it losing focus is a long known problem with any menu. Might be fixed with 516 since in that byond moves html renders to chromium

oh so not a suit sensors issue i see

**Comment by nikothedude** [comment]
closing as dupe of another issue

**Comment by Majkl-J** [comment]
Keep this open this is one of the few things that "Might be fixed in 516" so I'd like to track it

**Comment by projectkepler-ru** [comment]
> did you know tgui runs on internet explorer? Anyhow it losing focus is a long known problem with any menu. Might be fixed with 516 since in that byond moves html renders to chromium

hasn't it been swapped to edge? I heard there were plan to move it over to chromium at some point

**Comment by lessthnthree** [comment]
not yet, still ie

**Comment by Shadow-Quill** [comment]
Does this still happen with the 516 migration? Can't say I've noticed it happening anytime recently.
